packages/e2e-tests/src/specs/editor/fontCheck/fontCheck.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  withExperimentalFeatures,
+  createNewStory,
+  visitAdminPage,
+  insertStoryTitle,
+  publishStory,
+  takeSnapshot,
+  visitSettings,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Font Check', () => {
+  beforeEach(() => () => {});
+
+  afterEach(async () => {});
+
+  describe('Font Check Dialog', () => {
+    withExperimentalFeatures(['customFonts']); // 'notifyDeletedFonts'
+
+    it('should show dialog for missing font', async () => {
+      // add custom font
+      await visitSettings();
+      await page.waitForSelector('#insertFontUrl');
+      await page.type(
+        'input#insertFontUrl',
+        'https://overpass-30e2.kxcdn.com/overpass-regular.ttf'
```

Comment:
Is there a "long term" URL can pass here?

Reply:
Why not upload a font to the Docker container itself? :-)

For example, could download Open Sans Condensed Light from https://github.com/google/fonts/tree/d4ace626a71b0bc5def793af97e42bcfe49c4aae/apache/opensanscondensed into `packages/e2e-tests/src/assets`.

Then, add `wp media import /var/www/html/wp-content/e2e-assets/OpenSansCondensed-Light.ttf --quiet` here:

https://github.com/GoogleForCreators/web-stories-wp/blob/66ccea4a623167c3849e71ba8b6842766c72696b/bin/local-env/install-wordpress.sh#L207-L211

After that, the file will be available under `http://localhost:8899/wp-content/....`.

The base URL is available as `process.env.WP_BASE_URL`, so you can use that to assemble the URL.

That should do the trick.